tests/github-action-helper.sh

```diff
@@ -99,6 +144,11 @@ wait_for_operator_pod_to_be_ready_state_default() {
       sleep 1
     done
 EOF
+  # timeout command return exit status 124 if command times out
+  if [ $? -eq 124 ]; then
```

Comment:
define const for 124 and use it as its used in different places

Reply:
since we created helper method and `124` is used once so I skipped creating const